src/js/components/ResultItem.jsx

```diff
@@ -51,9 +51,6 @@ export class ResultItem extends Component {
             const statusGlyph = getGlyphFor(resultClean);
             this.setState({resultClean, statusGlyph});
         }
-        if (props.expanded!=this.props.expanded) {
```

Comment:
removing this will cause the callbacks not to fire: `onExpand` & `onCollapse`, that seems like a :bug: